include/xrpl/beast/utility/instrumentation.h

```diff
@@ -28,37 +28,43 @@ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 #endif
 #include <antithesis_sdk.h>
 #else
-#define ALWAYS(cond, name, ...) assert((name) && (cond))
-#define ALWAYS_OR_UNREACHABLE(cond, name, ...) assert((name) && (cond))
-#define SOMETIMES(cond, name, ...)
-#define REACHABLE(name, ...)
-#define UNREACHABLE(name, ...) assert((name) && false)
+// Macros below are copied from antithesis_sdk.h and slightly simplified
+// The duplication is because Visual Studio 2019 cannot compile this header
+// even with the option -Zc:__cplusplus added.
```

Comment:
When you say "this header", are you referring to antithesis_sdk.h? If so, I think "that header" would be more correct.